app/views/restaurants/show.html.erb

```diff
@@ -1,8 +1,15 @@
-<%= @restaurant.name %>
+Restaurant: <%= @restaurant.name %>
 
 
 <% @dishes.each do |dish| %>
     <%= dish.title %>
     <%= dish.description %>
     <%= dish.price %>
-<% end %>
\ No newline at end of file
+
+    <%= form_tag(orders_path) do %>
+      <%= submit_tag("Add", id: dish.id) do %>
+
+      <% end %>
+      <%= hidden_field_tag :dish_id, dish.id %>
```

Comment:
For better readability I would first add the hidden field and then the submit button.